src/navigation/AppNavigator.tsx

```diff
@@ -22,7 +21,7 @@ const AppNavigator = () => {
   };
 
   return (
-    <NavigationContainer linking={linking} fallback={<Text>Loading...</Text>}>
+    <NavigationContainer linking={linking}>
```

Comment:
Curious: what was happening while the `Loading...` text was displayed ? fetching the member ? refreshing the token ? 

I am not sure making the SpashScreen last longer would be the best UX. Having a nice welcome screen as the fallback would maybe increase the sentiment of responsiveness and lower the frustration while welcoming the user into the app. What do you think ?

Reply:
While the Loading... message was shown the app was loading the React Navigation. So, there were 3 main phases of loading the app:
1. `App.tsx` (initiating `AuthProvider`, `QueryClient`...) - it was displaying the splash screen
2. `React Navigation` (`Navigation Container`) - it had its own fallback (the Loading... text). The default one is a blank screen.
3. The rest of the nested components in `AppNavigator.tsx` (`CurrentMemberProvider`, `BottomSheetModalProvider`...) until the first RootNavigator, where the first actual content is displayed.

For 2. I followed the docs, which state to use the splash screen instead of the fallback (https://reactnavigation.org/docs/navigation-container/#fallback) and maybe hide the splash screen after the navigation is ready (https://reactnavigation.org/docs/navigation-container/#onready).

I tested it hiding the splash screen on the onReady React Navigation fallback, but it still displays a little blank screen (I suppose from the 3 point). Then I followed the docs of RN splash screen (https://docs.expo.dev/versions/latest/sdk/splash-screen/#usage) that recommends using the splash screen until the real content is correctly displayed. In our case I believe is once the `RootNavigator` is correctly mounted.

In case we do not want to display the same splash screen until the end, what do you think we should implement? A skeleton view?